renderer/index.js

```diff
@@ -458,7 +458,8 @@ function setVolume (volume) {
   if (isCasting()) {
     Cast.setVolume(volume)
   } else {
-    state.playing.setVolume = volume
```

Comment:
I think this is the way to dispatch that you need to change the volume to the <video> tag . 
Also, this is only for the local renderer and not the cast devices